src/subscription_manager/exceptions.py

```diff
@@ -149,6 +156,18 @@ def format_rate_limit_exception(self, rate_limit_exception, _):
         else:
             return RATE_LIMIT_MESSAGE
 
+    def format_cert_loading_error(self, exc: CertificateLoadingError, _: str):
+        message = exc.args[0]
+        file = exc.args[1]
+        pem = exc.args[2]
```

Comment:
You can see... These indexes used for accessing strings inside tuple are just cryptic.